index.html

```diff
@@ -0,0 +1,194 @@
+            </div>
+          </div>
+        </nav>
+      </div>
+
+      <div class="showcase index">
+        <blockquote>&ldquo; Hello! Fashion Peeps &rdquo;</blockquote>
+        <h1>CREATIVE STITCHES GLOBAL CONFERENCE 2021</h1>
+
+        <div class="para-border">
+          <p>
+            A great celebration for the fashion like minds, where we share tip
+            and open your heart to new insight of
+            <span class="highlight">fashion </span>
+          </p>
+        </div>
+        <div class="venue">
+          <h2>2021.10.15(THU) ~ 15(SAT)</h2>
+          <p>@ National park Lagos Nigeria.</p>
+        </div>
+      </div>
+    </header>
+
+    <main>
+      <section class="main">
+        <div class="program">
+          <div class="main-heading">
+            <h1>Main Program</h1>
+            <div class="sm-line"></div>
+          </div>
+          <section class="program-list">
+            <article class="li">
+              <img src="./images/lecture.png" alt="fashionicons" />
+              <h4>Lecture</h4>
+              <p>
+                We teach you, what it takes to know about fashion and be
+                fashionable. You can proudly pick colors and trend
+              </p>
+            </article>
+            <article class="li">
+              <img src="./images/face.png" alt="fashionicons" />
+              <h4>Men Fashion Exhibition</h4>
+              <p>
+                Men fashin is its own unique style, Talks on cloths that brings
+                the calm and bold you.
+              </p>
+            </article>
+            <article class="li">
+              <img src="./images/lady.png" alt="fashionicons" />
+              <h4>Women Fashion Exhibition</h4>
+              <p>
+                Learn to dress and look elegant as a woman, fly your colors
+                correctly
+              </p>
+            </article>
+            <article class="li">
+              <img src="./images/ropes.png" alt="fashionicons" />
+              <h4>The Fashion Way</h4>
+              <p>Discussions on the trend and hw to make it better</p>
+            </article>
+            <article class="li">
+              <img src="./images/tie.png" alt="fashionicons" />
+              <h4>Tie The knot</h4>
+              <p>
+                Rap up with communication and networking, get to meet leading
+                fashion tycoon around the world
+              </p>
+            </article>
+          </section>
+        </div>
+        <div class="summit">
+          <a href="#" class="hide-on-desktop">
+            Join Touch-Stiches Summit 2021</a
+          >
+          <a href="#" class="hide-on-mobile">See the whole program</a>
+        </div>
+      </section>
+      <section class="feature-speaker">
+        <div class="feature-heading">
+          <h1>Featured Speakers</h1>
+          <div class="sm-line"></div>
+        </div>
+        <div id="speak" class="speakers"></div>
+      </section>
```

Comment:
### OPTIONAL 
_I make it as a optional but these are requirements that missing. So please fix them for make it better project._

- [ ] In original mobile version speakers section has  3 speakers and this section has a `more button`. You can also do it with JavaScript. Also check the pictures width and height and background image.
### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/146978317-e91feb23-4c36-4b37-8c78-1aed632a8b00.png)
### YOUR DESIGN
![image](https://user-images.githubusercontent.com/38635158/146978372-6eb5139d-641a-4db1-882c-95955443c5b8.png)